fishtest/fishtest/rundb.py

```diff
@@ -531,6 +536,9 @@ def sync_update_task(self, run_id, task_id, stats, nps, spsa, username):
         self.stop_run(run_id, run)
         flush = True
 
+    seconds = int(time.time()) & 63
```

Comment:
Why not `% 60` ?

Reply:
It was just a way of trying to make this part of the code very simple and quick, on the assumption that the views are a lot less frequent than this part, so I do the ` * 60.0 / 64.0 ` calculation there. But maybe ` % 60 ` here, just an int operation, and then no calc at all required in the view is better - I can do it that way if you prefer.

Reply:
Speed will be the same for & and % in Python, so % is preferred.